tutorpicasso/commands/enable_themes.py

```diff
@@ -25,16 +26,29 @@ def enable_themes() -> None:
     # because it comes from the Tutor framework.
     # We are not handle type errors related to this object.
     for theme in tutor_conf["PICASSO_THEMES"]:  # type: ignore
-        if not {"name", "repo", "version"}.issubset(theme.keys()):  # type: ignore
+        if not isinstance(theme, dict):
             raise click.ClickException(
-                f"{theme} is missing one or more required keys: "
-                "'name', 'repo', 'version'"
+                "Expected 'theme' to be a dictionary, but got something else."
             )
 
-        theme_path = f'{tutor_root}/env/build/openedx/themes/{theme["name"]}'  # type: ignore
-        if os.path.isdir(theme_path):
-            subprocess.call(["rm", "-rf", theme_path])
-
-        subprocess.call(
-            ["git", "clone", "-b", theme["version"], theme["repo"], theme_path],  # type: ignore
-        )
+        else:
```

Comment:
You don't need the else here, since `if not isinstance(theme, dict)` evaluates to True, then the execution will stop altogether. So these lines won't be executed either way. Could we remove it so it's easier to read?